perf/lua/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(TARANTOOL_BIN $<TARGET_FILE:tarantool>)
+set(RUN_PERF_LUA_TEST "")
+
+function(create_perf_lua_test)
+  cmake_parse_arguments(
+    PERF
+    ""
+    "NAME"
+    ""
+    ${ARGN}
+  )
+  message(STATUS "Creating Lua performance test ${PERF_NAME}_perftest")
+  add_custom_target(${PERF_NAME}_perftest
+                    COMMAND ${TARANTOOL_BIN} ${CMAKE_CURRENT_SOURCE_DIR}/${PERF_NAME}.lua
+                    COMMENT Running ${PERF_NAME}_perftest
+                    DEPENDS tarantool
```

Comment:
Minor: It's worth adding Lua sources to `DEPENDS` section either.

Reply:
Updated:

```diff
--- a/perf/lua/CMakeLists.txt
+++ b/perf/lua/CMakeLists.txt
@@ -10,10 +10,11 @@ function(create_perf_lua_test)
     ${ARGN}
   )
   message(STATUS "Creating Lua performance test ${PERF_NAME}_perftest")
+  set(TEST_PATH ${CMAKE_CURRENT_SOURCE_DIR}/${PERF_NAME}.lua)
   add_custom_target(${PERF_NAME}_perftest
-                    COMMAND ${TARANTOOL_BIN} ${CMAKE_CURRENT_SOURCE_DIR}/${PERF_NAME}.lua
+                    COMMAND ${TARANTOOL_BIN} ${TEST_PATH}
                     COMMENT Running ${PERF_NAME}_perftest
-                    DEPENDS tarantool
+                    DEPENDS tarantool ${TEST_PATH}
                     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
   )
   set(RUN_PERF_LUA_TESTS_LIST ${RUN_PERF_LUA_TESTS_LIST} ${PERF_NAME}_perftest PARENT_SCOPE)
```